.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.3
```

Comment:
as the gem targets several rubies (see .travis.yml) we should not include a .ruby-version file.